pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[project]
+name = "curl-fuzzer-tools"
+version = "0.1.0"
+description = "Tooling for the curl-fuzzer repository"
+authors = [
+    {name = "Max Dymond",email = "cmeister2@gmail.com"}
+]
+readme = "README.md"
+requires-python = "^3.9"
```

Comment:
I think this is poetry-specific syntax.

Reply:
https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#python-requires

Appears to be standard.

Reply:
The field is, but I don't think the carat is. But correct me if I'm wrong.